src/app/products/components/button/button.component.html

```diff
@@ -0,0 +1,3 @@
+<button [ngClass]="classes" class="button" type="button">
```

Comment:
Better to define classes inside the component, not to pass them as an input parameter.
If button thould looks completely different, better way to create theme (also with defined classes inside the component), or create another button component